cypress/page/shipping-step.page.ts

```diff
@@ -0,0 +1,19 @@
+class ShippingStepPage {
+    private termsCheckbox: string;
+    private checkoutButton: string;
+
+    constructor() {
+      this.termsCheckbox = "#cgv";
+      this.checkoutButton = "[name='processCarrier']";
+    }
+  
+    public acceptTerms(): void {
+        cy.get(this.termsCheckbox).check();
+    }
+
+    public proceedToCheckoutIV(): void {
+        cy.get(this.checkoutButton).click();
+    }
```

Comment:
There's no need to call as "IV"
```suggestion
    public proceedToCheckout(): void {
        cy.get(this.checkoutButton).click();
    }
```